dashboards/traefik/traefik.json

```diff
@@ -78,7 +78,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "FROM Metric SELECT\nsum(traefik_entrypoint_requests_total) AS 'Total Requests'\nFACET instance, entrypoint\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write'"
+                "query": "FROM Metric SELECT\nsum(traefik_entrypoint_requests_total) AS 'Total Requests'\nFACET instance, entrypoint\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write' AND prometheus_server IN ( {{prometheus_server}}) AND instance IN ( {{instance}} ) facet concat(prometheus_server,' - ',instance)"
```

Comment:
2 `FACET` conditions in this query; recommend consolidating into:

```
FACET concat(prometheus_server,' - ',instance) AS 'Instance', entrypoint
```

Reply:
Thanks, changes made.